### DISCUSSION TITLE
new book addition

### DISCUSSION THREAD

**Discussion by Norventwire** [kinkofer/FightClub5eXML#181] [General]
Are there any plans to add Strixhaven: A Curriculum of Chaos to the compendium?

**Comment by zcdziura** [comment]
I'm not sure when @kinkofer is going to update this repository, or approve some of the pending Pull Requests to update data.

At any rate, you can check out the repository below to convert the Strixhaven data from 5e.tools to Fight Club 5e importable files:

https://github.com/rrgeorge/5eTtoFC5
